<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1678</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YUV420 Analyser By Lars C. Schwensen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Reference Video</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="referenceTypeComboBox">
             <item>
              <property name="text">
               <string>Full frame</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Luminance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chroma U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chroma V</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe luminance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe chroma U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe chroma V</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Intraframe luminance 2.5%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>intraframe chroma U 2.5%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>intraframe chroma V 2.5%</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="videoViewReferenceGraphicsView">
          <property name="minimumSize">
           <size>
            <width>818</width>
            <height>618</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>818</width>
            <height>618</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Processed Video</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="processedVideoComboBox">
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="processedTypeComboBox">
             <item>
              <property name="text">
               <string>Full frame</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Luminance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chroma U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chroma V</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe luminance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe chroma U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interframe chroma V</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Intraframe luminance 2.5%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Intraframe chroma U 2.5%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Intraframe chroma V 2.5%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mean square error luminance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mean square error chroma U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mean square error chroma V</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="videoViewProcessedGraphicsView">
          <property name="minimumSize">
           <size>
            <width>818</width>
            <height>618</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>818</width>
            <height>618</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="saveReferencePushButton">
        <property name="text">
         <string>Save to file!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveProcessedPushButton">
        <property name="text">
         <string>Save to file!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zoomHorizontalSlider">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Frame Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>299</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="framesAvailableProgressBar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>7</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSpinBox" name="framesSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="firstFramePushButton">
            <property name="text">
             <string>First Frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPushButton">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopPushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Infiormation</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Frame:</string>
         </property>
        </widget>
        <widget class="QLabel" name="nrFramesLabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>No Info</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>MSE:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>PSNR:</string>
         </property>
        </widget>
        <widget class="QLabel" name="mseLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>65535</string>
         </property>
        </widget>
        <widget class="QLabel" name="psnrLabel">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>60</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>135 dB</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1678</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
